pyccel/ast/core.py

```diff
@@ -1340,7 +1340,7 @@ class Iterable(Basic):
     iterable : acceptable_iterator_type
                 The iterator being wrapped
     """
-    indexless_types = (PythonRange, PythonEnumerate)
+    # Types for which an extra index does not need to be provided
```

Comment:
Thanks for adding this comment line! But I think it's obsolete now, after the latest changes...

Reply:
Fixed 3b636b5